### DISCUSSION TITLE
Downloaded New Version - Does not Burn!

### DISCUSSION THREAD

**Discussion by imagecatcher01** [arkypita/LaserGRBL#891] [Q&A]
I installed the new version, rebooted compter...it does not burn!  The light for the laser is on, but 
does not burn the wood.

**Comment by arkypita** [comment]
I have done some changes in the last version of LaserGRBL, but nothing that can justify this behavior. To be sure that LaserGRBL is commanding to use the max power check your grbl parameters and confirm that are like this:

![image](https://user-images.githubusercontent.com/8782035/75434118-5d413900-5951-11ea-8ce4-31f279210f3b.png)

Then try to engrave something (a simple logo, with vectorization, no filling) with a low border speed and in "Laser Options" set 0 - 255 as min/max.

![image](https://user-images.githubusercontent.com/8782035/75434597-17d13b80-5952-11ea-811f-650264ba9e2a.png)


Usually the fact that the laser turns on with a weak light that does not burn is a symptom of a "dead" diode. https://github.com/arkypita/LaserGRBL/issues/780

Please note that LaserGRBL does not physically have the possibility of breaking the diode (least of all with an update!).

If the diode stop working it is because it is a faulty one or because it is a poor quality one supercharged with higher powers than it can truly support. 

https://www.youtube.com/watch?v=81Io-ghEIgk

**Comment by imagecatcher01** [comment]
Thanks, I will try it again with the new version.  I know it is not a dead
diode because it still burns with the previous version.

Dave Hupke

**Comment by arkypita** [comment]
> Thanks, I will try it again with the new version. I know it is not a dead diode because it still burns with the previous version.

Good, I am really happy for you!

In the latest version I have included some changes that have been sent to me by other collaborators, to include support for firmware other than grbl (especially smoothie and marlin). These changes should not have affected Grbl.

Are you using Smootie or Marlin firmware accidentally? Let me know please.

![image](https://user-images.githubusercontent.com/8782035/75464247-39004f00-5987-11ea-8557-36c78cf74607.png)

**Comment by arkypita** [comment]
Can you check it in your settings please?

**Comment by k-str50** [comment]
@arkypita 
first of all I want to determine that LaserGRBL seems to be a fine programm.
But I'm sorry I cannot use it.
How can I get this window? Must I connect the laser machine do my computer?
![75434597-17d13b80-5952-11ea-811f-650264ba9e2a](https://user-images.githubusercontent.com/57534611/118352604-a5527880-b562-11eb-8e24-8170d60146b8.png)
My Laser engraver is an upgraded Ender 3 pro.

**Comment by StuartB4** [comment]
"How can I get this window? Must I connect the laser machine do my computer?"

Start LaserGRBL. Load an image adjust any settings you need to change and click Next. 
The box in your image above should appear.

**Comment by StuartB4** [comment]
If you mean the Settings window that arkypita posted then you click the Grbl menu and Settings.

**Comment by k-str50** [comment]
@StuartB4 ,
Thanks for your answer.
Yes thats what I mean.
Must I connect the PC to the machine?
When I Open the GRBL-Window it's empty!
Does it only work if the machine is connected?
![GRBL](https://user-images.githubusercontent.com/57534611/118359402-83b5b900-b583-11eb-8299-f04f89bee93f.JPG)
**What must I do if I want to write the gcode to a SD-Card?**

**Comment by StuartB4** [comment]
Even if you don't connect the list of settings should be there when you click grbl configuration.
You only need to be connected if you want to change any of the settings.

**Reply by arkypita** [reply]
Right, but only after the first valid connection

**Comment by k-str50** [comment]
@arkypita , @StuartB4 
thanks to your answers.
I was working on my problem. And I'm found out that I have to set the firmware type to **marlin**. Then I can generate gcode and I can connect the 3D printer to my PC.
The laser module is driven by th cooler connector.
It works quiet fine but the laser doesn't work.
I think I have to modify the gcode for **laser** to the **fan** to **M106 = laser ON** and **M107 = Laser OFF**. Is there any place in the software where I can set this?

**Reply by k-str50** [reply]
Hi,
now I could connect to the laser engraver Ender 3 pro with laser module.
But I cannot modify the laser command for target image.

1. I set the protocol to marlin
2. loading a picture (=jpg-file)
3. click on continie (weiter)
4. here I get only the possibillity to select laser mode M3 constant power or M4 dynamic power.
What do I wrong?
![grafik](https://user-images.githubusercontent.com/57534611/122726708-8be5d000-d276-11eb-9617-368490e84530.png)
![grafik](https://user-images.githubusercontent.com/57534611/122726907-c2234f80-d276-11eb-8518-b6f3af91a0c7.png)
![grafik](https://user-images.githubusercontent.com/57534611/122726953-cfd8d500-d276-11eb-80fe-a99bd65d21ee.png)
